build_projects/shared-build-targets-utils/Utils/PublishMutationUtilties.cs

```diff
@@ -22,7 +22,7 @@ public class PublishMutationUtilties
             foreach (var binaryName in binariesToBeDeleted)
             {
                 File.Delete(Path.Combine(path, $"{binaryName}{Constants.ExeSuffix}"));
-                File.Delete(Path.Combine(path, $"{binaryName}.dll"));
+                File.Delete(Path.Combine(path, $"{Constants.DynamicLibPrefix}{binaryName}{Constants.DynamicLibSuffix}"));
```

Comment:
Just for informational purposes, I believe this method used to be used for managed assemblies, which is why the `{binaryName}.dll` entry was in there.